src/runner_test.go

```diff
@@ -35,55 +30,100 @@ vars:
         echo "$RHC_WORKER_FOO $RHC_WORKER_BAR!"
     content_vars:
         FOO: Hello
-        BAR: World`)
-	expectedResult = "Hello World!\n"
-	result = processSignedScript(yamlData)
-	if result != expectedResult {
-		t.Errorf("Expected %q, but got %q", expectedResult, result)
+        BAR: World`),
+			expectedResult: "Hello World!\n",
+		},
+		{
+			name:       "verification enabled, invalid signature = error msg returned",
+			verifyYAML: true,
+			yamlData: []byte(`
+vars:
+    _insights_signature: "invalid-signature"
+    _insights_signature_exclude: "/vars/insights_signature,/vars/content_vars"
```

Comment:
Small suggestion here, but I think that the names here should be without the `_` 

```suggestion
    insights_signature: "invalid-signature"
    insights_signature_exclude: "/vars/insights_signature,/vars/content_vars"
```

Reply:
fixed everywhere